src/test/java/com/endava/internship/mocking/service/PaymentServiceTest.java

```diff
@@ -1,31 +1,108 @@
 package com.endava.internship.mocking.service;
 
-import org.junit.jupiter.api.BeforeEach;
+import com.endava.internship.mocking.model.Payment;
+import com.endava.internship.mocking.model.Status;
+import com.endava.internship.mocking.model.User;
+import com.endava.internship.mocking.repository.PaymentRepository;
+import com.endava.internship.mocking.repository.UserRepository;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
 
+import java.util.Arrays;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.Optional;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
 @ExtendWith(MockitoExtension.class)
 class PaymentServiceTest {
 
+    @Mock(name = "userRepositoryMock")
```

Comment:
Name of a Mock appears in trace if test fails. Without it would be just smth like `Mock for Payment, hashCode: 480184864`. I  suppose that name is more suitable than default output.